e2e-tests/test/anvil-apis.test.ts

```diff
@@ -90,19 +91,29 @@ describe("anvil_setBlockTimestampInterval & anvil_removeBlockTimestampInterval",
 
 describe("anvil_setLoggingEnabled", function () {
   it("Should disable and enable logging", async function () {
+    const logFilePath = process.env.ANVIL_LOG_PATH || path.resolve("../anvil-zksync.log");
+
     // Arrange
     const wallet = new Wallet(RichAccounts[0].PrivateKey, provider);
     const userWallet = Wallet.createRandom().connect(provider);
 
     // Act
     await provider.send("anvil_setLoggingEnabled", [false]);
-    const logSizeBefore = fs.statSync("../anvil-zksync.log").size;
+
+    let logSizeBefore = 0;
+    if (fs.existsSync(logFilePath)) {
+      logSizeBefore = fs.statSync(logFilePath).size;
+    }
```

Comment:
Is this `if` necessary? Judging by the rest of the changes I think the log file will be present regardless of the running mode